### PR TITLE
Added styles for Input and Dropdown Box

### PR DESCRIPTION
**Pull request by GREEDnim**
### input box
image
### dropdown v1
image
### dropdown v2
image

### input box
image
### dropdown v1
image
### dropdown v2
image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -18,4 +18,42 @@
...
+  --logo-small:20px;
+
+  --dropdown-box-shadow:0px 4px 12px rgba(15,15,15,0.08);
+  --dropdown-v2-image-height:72px;
+
+  --dropdown-v2-gap:24px;
+}
+
+
+.w-400{
```
Comment (hariFED) [inline style.css]: Add all width classes to the root and use it in the respective component/parent class.

File: style.css [hunk 2]
```diff
@@ -18,4 +18,42 @@
...
+  width: 400px;
+}
+.w-300{
+  width: 300px;
+}
+.h-20{
+height:20px;
+}
+.px-12{
```
Comment (hariFED) [inline style.css]: Add all padding classes to the root and use it in the respective component/parent class.

File: Input.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
+/*  input box, dropdown box */
+
+
+.drop-box-shadow{
+box-shadow: var(--dropdown-box-shadow);
+}
+
+  
+.container{
```
Comment (aravind-digitech) [inline Input.css]: use naming convention (eg. drop-box-container)

File: Input.css [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    width:var(--logo-small);
+}
+
+.logo-field-container .field{
+    border: none;
+    outline: none;
+    flex-grow: 1;
+}
+
+.dv1{
```
Comment (aravind-digitech) [inline Input.css]: use naming convention (eg. drop-box-container) for all subsequent class names.

**Review by aravind-digitech — CHANGES_REQUESTED** [review]
pls comment once done

**Pull request closed without merging**